initfiles/examples/embed/couchbase-simple.ecl

```diff
@@ -0,0 +1,112 @@
+IMPORT couchbase;
+/******************************************************************************************
+   This set of example queries are based on the couchbase sample bucket 'travel-sample'
+   The sample data is easily set-up, pelase refer to the couchbase documentation for set-up
+*******************************************************************************************/
+
```

Comment:
I have couple of comments:

1. Because this test is a plugin/3rd party server test you should insert two tags into this file (practically here):
//class=embedded
//class=3rdparty

This tags used by the OBT/Smoketest engines to include/exclude those test based on the configuration of the environment.

2. Actually Smoketest doesn't configured to test plugins/3rdprty servers (except memcached which is [unfortunately] integrated into ESP server), but the OBT does.

3. I need information about expectations towards/configuration of Couchbase server if they are differs from out-of-box installation.